config/surf_snellius.py

```diff
@@ -34,16 +28,10 @@
                     'name': 'gpu',
                     'scheduler': 'slurm',
                     'prepare_cmds': ['source /cvmfs/pilot.eessi-hpc.org/latest/init/bash'],
-                    'launcher': 'srun',
+                    'launcher': 'mpirun',
```

Comment:
the launcher is changed to `mpirun` but with auto detect being true, it should be `srun`. right?

Reply:
I did go through the whole file. What I meant is, since the auto-detect is set
to `True`, if someone just clones this and starts running it out of box on
Snellius which is also a stretch, is it fine? Or are they NOT supposed to run
it out of box?

Reply:
Correct, it will not run out of the box _the first time_ they run this. But, since the auto-detect is a one time thing (as long as the partition names stay the same), I don't think that's a terrible issue. The next time that you clone the repo, and run it, it'll work, because the topology files are in you homedir (in `.reframe/topology`).

Reply:
Since it is clear, that it will not run out of the box, I will merge this for now. I have created an issue #73 for this.